staging/src/k8s.io/kubectl/pkg/describe/versioned/describe_test.go

```diff
@@ -3181,16 +3181,15 @@ Events:              <none>` + "\n"
 func TestDescribeNode(t *testing.T) {
 	fake := fake.NewSimpleClientset(&corev1.Node{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "bar",
-			Namespace: "foo",
+			Name: "bar",
 		},
 		Spec: corev1.NodeSpec{
 			Unschedulable: true,
 		},
 	})
 	c := &describeClient{T: t, Namespace: "foo", Interface: fake}
 	d := NodeDescriber{c}
-	out, err := d.Describe("foo", "bar", describe.DescriberSettings{ShowEvents: true})
+	out, err := d.Describe("", "bar", describe.DescriberSettings{ShowEvents: true})
```

Comment:
I'm not sure I'm following how this change relates to your other changes - would you mind adding some additional context?

Thanks :)

Reply:
Me neither anymore. Unit tests pass even without the change. Undone.